README.md

```diff
@@ -34,6 +34,28 @@ pnpm dev
 # http://localhost:3333
 ```
 
+## docker
+
+You can either choose to pull the image from docker hub or build your own one locally.
```

Comment:
Considering that this is a project that is designed for Chinese, perhaps we can use Chinese to describe it directly?